### DISCUSSION TITLE
FIXED: Mediamodule.m -> PHPickerDelegate don't call pickerdismiss

### DISCUSSION THREAD

**Discussion by mbender74** [appcelerator/titanium_mobile#13046] [General]
can be fixed with these lines:

```
#pragma mark PHPickerViewControllerDelegate


**- (void)presentationControllerDidDismiss:(PHPickerViewController *)picker
{
    [self sendPickerCancel];
}**

- (void)picker:(PHPickerViewController *)picker didFinishPicking:(NSArray<PHPickerResult *> *)results

```

**Comment by m1ga** [comment]
you can use
```
  \```
  code
  \```
```
(without the `\`) to format the code better so it will look better.

And perhaps it would be better to explain a bit more what the error and fix is or even add a PR (just try it :wink:  ).

**Reply by mbender74** [reply]
I promise, I will do better in the future ... ;-)
PR could be done .... but for now a JIRA ticket is opened for that already ;-)
https://jira.appcelerator.org/browse/AC-6716

**Comment by m1ga** [comment]
added the PR for you: https://github.com/appcelerator/titanium_mobile/pull/13057
so it won't get lost.
